test/api/worker/facades/LoginFacadeTest.ts

```diff
@@ -456,10 +450,14 @@ o.spec("LoginFacadeTest", function () {
 				when(restClientMock.request(matchers.contains("sys/session"), HttpMethod.GET, anything()))
 					.thenResolve(JSON.stringify({user: userId, accessKey: keyToBase64(accessKey)}))
 
-				await facade.retryAsyncLogin()
+				//We need to wait a bit so asyncResumeSesssion has ended - otherwise this will not be executed
+				setTimeout(async function () {
+					await facade.retryAsyncLogin()
+				}, 10);
```

Comment:
the promise for asyncResumeSession doesn't get returned anywhere so we could await it?

Reply:
Yes, we need `await delay(10)` instead

Reply:
would that ensure the fire-and-forget promise of asyncResumeSession that's started in resumeSession is done?